styles/cspace-ui/BannerMain.css

```diff
@@ -6,7 +6,6 @@
 
 .common > :global(.cspace-ui-Logo--common) {
   flex: 0 1 auto;
-  overflow: hidden;
```

Comment:
I don't recall why this was here. I think removing it is the best fix, and hopefully it won't break anything. If it does, another option would be to add `padding: 1px`, so that there is space around the `a` for the focus outline to be visible.

Reply:
I tried to check around the ui as best as I could and couldn't break anything. The banner only holds a few components anyway so everything should be ok.